dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -33,6 +33,10 @@ internal class RumActionScope(
 
     internal val ongoingResourceKeys = mutableListOf<WeakReference<Any>>()
 
+    internal var resourcesCount: Int = 0
+    internal var viewTreeChangeCount: Int = 0
```

Comment:
I don't see the usage if a counter here . Why not using only a flag ?

Reply:
Well, I thought about that, but in the end decided this was more compliant with futur evolution. Besides it doesn't cost more to use a counter over a boolean